lib/yocaml_syndication/yocaml_syndication.ml

```diff
@@ -0,0 +1,29 @@
+module Atom = struct
+  type t = Syndic.Atom.feed
+
+  let make =
+    Syndic.Atom.feed
+      ~generator:
+        { version = Some "dev"
+        ; uri = Some (Uri.of_string "https://github.com/xhtmlboi/yocaml")
+        ; content = "YOcaml"
+        }
```

Comment:
Are you sure about those hardcoded parameters? WDYT about parametrizing them using default labelled arguments?

Reply:
Fixed.